package.json

```diff
@@ -1,47 +1,77 @@
+  "description": "Allows to use Web Components using ES modules in your Ember app",
   "keywords": [
-    "ember-addon"
+    "ember-addon",
+    "web components",
+    "ES6 modules",
+    "bundle"
   ],
-  "repository": "",
+  "repository": "https://github.com/BBVAEngineering/ember-cli-webcomponents-bundler.git",
   "license": "MIT",
-  "author": "",
+  "author": "BBVAEngineering",
   "directories": {
-    "doc": "doc",
     "test": "tests"
   },
+  "files": [
+    "index.js",
+    "config/babel.js",
+    "config/output.js",
+    "config/rollup.js",
+    "utils/**"
+  ],
   "scripts": {
     "build": "ember build",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint --max-warnings 0 ./*.js addon app config tests",
+    "lint:js": "eslint --fix --max-warnings 0 ./*.js config tests",
     "start": "ember serve",
-    "test": "COVERAGE=true ember test",
-    "test:all": "ember try:each",
-    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 100 --functions 100 --branches 100 --lines 100",
+    "test": "npm-run-all test:*",
+    "test:ember": "COVERAGE=true ember test",
+    "test:node": "nyc mocha --recursive 'node-tests/**/*-test.js'",
+    "clean-coverage": "rm -rf coverage",
+    "merge-coverage": "cp coverage/ember/coverage-final.json coverage/ember.json && cp coverage/node/coverage-final.json coverage/node.json && nyc merge coverage coverage/coverage-final.json",
+    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 85 --functions 85 --branches 85 --lines 85",
     "commit": "git-cz",
-    "posttest": "npm run report-coverage && npm run check-coverage && npm run validate-coverage",
+    "posttest": "npm run merge-coverage && npm run report-coverage && npm run check-coverage",
     "report-coverage": "istanbul report --include=coverage/coverage-final.json text",
     "semantic-release": "semantic-release",
     "validate-coverage": "test \"$(cat coverage/coverage-summary.json | json total.lines.total)\" -gt 0"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.8.0"
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/plugin-syntax-import-meta": "^7.2.0",
+    "broccoli-funnel": "^2.0.2",
+    "broccoli-merge-trees": "^3.0.2",
+    "broccoli-rollup": "^4.1.1",
+    "ember-cli-babel": "^7.7.3",
+    "regenerator-runtime": "^0.13.2",
+    "rollup": "^1.15.6",
+    "rollup-plugin-babel": "^4.3.2",
+    "rollup-plugin-commonjs": "^10.0.0",
+    "rollup-plugin-includepaths": "^0.2.3",
+    "rollup-plugin-multi-entry": "^2.1.0",
+    "rollup-plugin-node-resolve": "^5.0.3",
+    "rollup-plugin-terser": "^5.0.0"
   },
   "devDependencies": {
     "@commitlint/cli": "^8.0.0",
     "@commitlint/config-conventional": "^8.0.0",
     "@ember/optional-features": "^0.7.0",
     "@semantic-release/changelog": "^3.0.4",
     "@semantic-release/git": "^7.0.12",
-    "broccoli-asset-rev": "^3.0.0",
+    "@webcomponents/webcomponentsjs": "^2.2.10",
```

Comment:
I think this must be a production dependency...

Reply:
I think that is a devDependency because it's used only for the demo, but I don't know if `addPackageToProject` requires this package to be included as an addon dependency.

Reply:
After checking another [projects using `addPackageToProject`](https://github.com/search?q=this.addPackageToProject&type=Code), I found that is not required to have the dependencies that will be included in the hosting application listed in the addon package.json, so it's fine to have the dependency as devDependency.

Reply:
Nice.